tests/integration/test_storage_kerberized_kafka/test.py

```diff
@@ -97,6 +97,19 @@ def test_kafka_json_as_string(kafka_cluster):
     assert instance.contains_in_log("Parsing of message (topic: kafka_json_as_string, partition: 0, offset: 1) return no rows")
 
 def test_kafka_json_as_string_no_kdc(kafka_cluster):
+    instance.query('''
```

Comment:
Worth adding a comment why this unused table is required (to create message in log).

Reply:
Ok.